examples/pico-examples/hello.py

```diff
@@ -25,7 +25,7 @@
 
 sm = rp2pio.StateMachine(
     assembled,
-    frequency=2000,
+    frequency=10000,
```

Comment:
Why change this? 2000 should work.

Reply:
It was reported in Learn feedback that the 2000 value didn't work.  Rather than investigate, I chose to bump the value.  I can test it, perhaps the feedback was wrong.

Reply:
you're right, this works.  I dropped it.  Please re-review @tannewt